src/parameters/parameters_reduced_order.cpp

```diff
@@ -28,7 +40,20 @@ void ReducedOrderModelParam::parse_parameters (dealii::ParameterHandler &prm)
     {
         adaptation_tolerance = prm.get_double("adaptation_tolerance");
         reduced_residual_tolerance = prm.get_double("reduced_residual_tolerance");
+        num_halton = prm.get_integer("num_halton");
         path_to_search = prm.get("path_to_search");
+
+        std::string parameter_names_string = prm.get("parameter_names");
+        std::unique_ptr<dealii::Patterns::PatternBase> ListPatternNames(new dealii::Patterns::List(dealii::Patterns::Anything(), 0, 10, ",")); //Note, in a future version of dealii, this may change from a unique_ptr to simply the object. Will need to use std::move(ListPattern) in next line.
```

Comment:
It could be, but I am not sure. I came across that on some website and thought it would be good to note for future reference. Not sure if it merits an issue, since it isn't currently an issue.

Reply:
Okay, sounds good!